airflow/gcp/operators/natural_language.py

```diff
@@ -62,8 +63,8 @@ class CloudLanguageAnalyzeEntitiesOperator(BaseOperator):
     def __init__(
         self,
         document: Union[dict, Document],
-        encoding_type: EncodingType = None,
-        retry: Retry = None,
+        encoding_type: Optional[enums.EncodingType] = None,
```

Comment:
I think previous version was more readable. So maybe it's worth to import `*.enums.EncodingType`?

Reply:
I did like that because as it in @mik-laj comments below, indicates the type at first glance. More code, but more information as well.

Reply:
It is also consistent with type annotations from corresponding hook.